apps/api/src/app/events/usecases/send-message/send-message-email.usecase.ts

```diff
@@ -119,7 +119,7 @@ export class SendMessageEmail extends SendMessageBase {
       return;
     }
 
-    const overrides = command.overrides[integration?.providerId] || {};
+    const overrides: Record<string, any> = command.overrides[integration?.providerId] || {};
```

Comment:
Aren't we taking the information from the "email" field and not from the providerId field? 

So when the user is sending a trigger he specifies it as:
```
novu.trigger('template', {
...data,
overrides: {
   email: { // <---- Here it's an email and not a provider Id 
     cc: []
   }
}
})
```

Reply:
I thought this was the wished structure....

Reply:
a good point but right now we expect this: 
```
novu.trigger('template', {
...data,
overrides: {
  providerId: {
     email: { // <---- Here it's an email and not a provider Id 
       cc: []
     }
  }
}
})
```
Will remove so email is not needed